pages/pipelines/command_step.md.erb

```diff
@@ -2,26 +2,142 @@
 
 <%= toc %>
 
-A *command* step runs a command on an agent. The following is the most simple *command* step:
+
+A *command* step runs one or more shell commands on an agent. Each command step can run either a shell command like `npm install`, or an executable file or script like `build.sh`.
+
+<%= image "pipeline-settings.png", width: 1158/2, height: 506/2, alt: "Screenshot of a basic command step" %>
+
+A basic command step can be defined in your pipeline settings, or in your [pipeline.yml](/docs/pipelines/uploading-pipelines) file.
+
+```yml
+steps:
+  - command: "tests.sh"
+```
+
+
+## Command Step Attributes
+
+_Required attributes:_
+
+<table>
+  <tr>
+    <td><code>command</code></td>
+    <td>
+      The shell command/s to run during this step. This can be a single line of commands, or a yaml list. <br>
+      <em>Example:</em> <code>"build.sh"</code><br>
+      <em>Example:</em> <code>"npm install && npm test"</code>
+    </td>
+  </tr>
+</table>
 
 ```yml
-- command: "command.sh"
+steps:
+  - command: "npm install && tests.sh"
 ```
 
-The following step includes every configuration property for *command* steps:
+```yml
+steps:
+  - command:
+      - "npm install && npm test" 
+      - "moretests.sh"
+      - "build.sh"
+```
+
+_Optional attributes:_
+
+<table>
+  <tr>
+    <td><code>name</code></td>
+    <td>
+      The label that will be displayed in the pipeline visualisation in Buildkite. (supports emoji)<br>
+      <em>Example:</em> <code>"\:hammer\: Tests"</code><br>
+    </td>
+  </tr>
+  <tr>
+    <td><code>branches</code></td>
+    <td>
+      The <a href="/docs/pipelines/branch-configuration#branch-pattern-examples">branch pattern</a> defining which branches will include this step in their builds.<br>
+      <em>Example:</em> <code>"master stable/*"</code>
+    </td>
+  </tr>
+  <tr>
+    <td><code>env</code></td>
+    <td>
+      A list of <a href="/docs/builds/environment-variables">environment variables</a> for this step. <br>
+      <em>Example:</em> <code>FOO: "bar"</code>
```

Comment:
This should probably include a multi-line example to showcase the format the ENV vars take:

```
steps:
  - command: "test.sh"
    env:
      RAILS_ENV: "test"
      RACK_ENV: "test"
```